core-processor/src/processor.rs

```diff
@@ -27,7 +27,7 @@ use gear_backend_common::Environment;
 use gear_backend_common::ExtInfo;
 use gear_core::{
     env::Ext as EnvExt,
-    message::{Dispatch, DispatchKind, Message},
+    message::{Dispatch, DispatchKind, ExitCode, Message},
```

Comment:
I would go with something other than "exit code". It might be numerated C-like enum, scale encodable to also be able to parse on the js side. But it may be done later

Reply:
Good Idea